inventory.py

```diff
@@ -213,7 +213,17 @@ def lambda_list(session):
                         last = "never"
                         invoked = "none"
 
-            lambdas.append([lambdaFunction['FunctionName'], session.profile_name, region, lambdaFunction['Runtime'], str(last), str(invoked) ])
+            if lambdaFunction['PackageType'] == 'Zip':
+                lambdas.append([lambdaFunction['FunctionName'], session.profile_name, region, lambdaFunction['Runtime'], str(last), str(invoked) ])
+            else:
```

Comment:
Is this just an either/or situation, or are there other cases besides zip or docker that should be taken into account?

Reply:
It's an either or situation.
https://docs.aws.amazon.com/lambda/latest/dg/gettingstarted-package.html